src/EventStore.Core/Services/Transport/Tcp/TcpPackage.cs

```diff
@@ -40,14 +53,25 @@ public struct TcpPackage {
 			string login = null;
 			string pass = null;
 			if ((flags & TcpFlags.Authenticated) != 0) {
-				var loginLen = data.Array[data.Offset + AuthOffset];
-				if (AuthOffset + 1 + loginLen + 1 >= data.Count)
-					throw new Exception("Login length is too big, it does not fit into TcpPackage.");
+				if (!BitConverter.IsLittleEndian) {
```

Comment:
```suggestion
				if (BitConverter.IsLittleEndian) {
```

Reply:
I think we don't need to check this at all actually